docs/user-guide/configuring.md

```diff
@@ -550,84 +550,125 @@ The complete configuration hierarchy, from highest precedence to lowest preceden
 
 ## Extending Configuration Files
 
-If you want to extend a specific configuration file, you can use the `extends` property and specify the path to the file. The path can be either relative or absolute.
+A configuration file can "extend" one or more "child" configurations to override a "base" set of rules.
 
-Configurations can be extended by using:
+The `extends` property value is either a string or an array of strings (in which each additional configuration overrides the preceding configurations) that can specify any of the following:
 
-1. YAML file
-1. JSON file
-1. JS file
-1. Shareable configuration package
+* the recommended rules
```

Comment:
I think this list makes it more confusing rather than clarifying anything. The important thing to know about `extends` is that it can be a string or an array of strings, and that each string represents a shareable config to inherit from. What those strings represent doesn't make sense at this point in the conversation.